doc/contribute/modifying_contributions.rst

```diff
@@ -0,0 +1,70 @@
+.. _modifying_contributions:
+
+Modifying Contributions made by other developers
+************************************************
+
+Scenarios
+#########
+
+In general, Zephyr contributors and collaborators are encouraged to assist
+as reviewers in pull requests, so that patches may be approved and merged
+to Zephyr's main branch as part of the original pull requests. The authors
+of the pull requests are responsible for amending their original commits
+following the review process.
+
+There are occasions, however, when a contributor might need to modify patches
+included in pull requests that are submitted by other Zephyr contributors.
+For instance, this is the case when:
+
+* a developer picks commits submitted by other contributors into their own
+  pull requests in order to:
+
+  * integrate useful content which is part of a stale pull request, or
+  * get content merged to the project's main branch as part of a larger
+    patch
+
+* a developer pushes onto a branch or pull request opened by another
+  contributor in order to:
+
+  * assist in revising pull requests and get the patches merged
+    to the project's main branch
+  * finalize review in stale pull requests
+
+
+Accepted policies
+#################
+
+A developer who intends to picking and modifying patches sent by
+another contributor shall:
+
+* clarify in their pull request the reason for doing that, instead
+  of assisisting in getting the patches merged in their original
+  pull request, and
```

Comment:
why instead of?

Reply:
Hm, I guess "instead of", here, is used to distinguish between "doing that" and "assisting in getting..." 
As it seems not to be clear, I'll replace "doing that" with the actual verb action :)

Reply:
fixed now